src/System.Management.Automation/help/HelpCommands.cs

```diff
@@ -273,17 +273,18 @@ protected override void ProcessRecord()
                 // Validate input parameters
                 ValidateAndThrowIfError(helpCategory);
 
-                HelpRequest helpRequest = new HelpRequest(this.Name, helpCategory);
-
-                helpRequest.Provider = _provider;
-                helpRequest.Component = Component;
-                helpRequest.Role = Role;
-                helpRequest.Functionality = Functionality;
-                helpRequest.ProviderContext = new ProviderContext(
+                HelpRequest helpRequest = new HelpRequest(this.Name, helpCategory)
+                {
+                    Provider = _provider,
+                    Component = Component,
+                    Role = Role,
+                    Functionality = Functionality,
+                    ProviderContext = new ProviderContext(
                     this.Path,
                     this.Context.Engine.Context,
-                    this.SessionState.Path);
-                helpRequest.CommandOrigin = this.MyInvocation.CommandOrigin;
+                    this.SessionState.Path),
```

Comment:
Identations.

Reply:
Seems there are bugs in IDE0017 code fix, I will open an issue in dotnet/roslyn